airbyte-workers/src/test/java/io/airbyte/workers/temporal/TemporalAttemptExecutionTest.java

```diff
@@ -32,24 +32,39 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-import io.airbyte.commons.functional.CheckedConsumer;
 import io.airbyte.commons.functional.CheckedSupplier;
+import io.airbyte.config.Configs;
+import io.airbyte.db.Database;
+import io.airbyte.db.instance.DatabaseMigrator;
+import io.airbyte.db.instance.jobs.JobsDatabaseInstance;
+import io.airbyte.db.instance.jobs.JobsDatabaseMigrator;
 import io.airbyte.scheduler.models.JobRunConfig;
 import io.airbyte.workers.Worker;
 import io.temporal.internal.common.CheckedExceptionWrapper;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.sql.SQLException;
 import java.util.function.Consumer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.mockito.stubbing.Answer;
+import org.testcontainers.containers.PostgreSQLContainer;
 
 class TemporalAttemptExecutionTest {
```

Comment:
I added a test in the acceptance tests.

I don't see why not. Will add one.

Reply:
Hum I tried to do this and realised since the cancellation handler polls temporal for to figure out if the job is cancelled, this is going to need a working temporal cluster to be a valuable test. Possible but don't think it's worth the effort given we have this in the acceptance tests.